### ISSUE TITLE
Listing 7.20 Custom Callback (2nd edition)

### ISSUE THREAD

**Issue by steval88** [fchollet/deep-learning-with-python-notebooks#222]
Dear Francois

there's a small typo in Listing 7.20 on page 190. In `model.fit(),` `epochs` is set to 10, but to be consistent with Figure 7.5, it should be `epochs=1`

Best

Stefano

**Comment by steval88** [comment]
I was double-checking : it's actually not a typo, but if you run Listing 7.20  on jupyter without changing the `epoch` number, it yields the last training loss at `epoch=9`. However, you can retrieve the training loss from the plots which are created and stored in your local folder for each `epoch`. Steady state loss  is around 0.07 at epoch=9 after 200-400 batches. Maybe adding some comments about the different behaviors (e.g. for small batch numbers for different `epoch` numbers)  would have been interesting
